openslides/core/apps.py

```diff
@@ -63,6 +66,13 @@ def ready(self):
         router.register(self.get_model('ProjectorMessage').get_collection_string(), ProjectorMessageViewSet)
         router.register(self.get_model('Countdown').get_collection_string(), CountdownViewSet)
 
+        # Sets the cache
+        try:
+            element_cache.ensure_cache()
+        except (ImproperlyConfigured, OperationalError):
```

Comment:
@normanjaeckel I don't like this but I don't know how this can be fixed. The OperationalError happens when migrations are run. In migrations, this code is run before the database is created. Therefore the database tables do not exist and the cache can not be created.

The ImproperlyConfigured error occurs in tests, where the database is deactivated. See: https://pytest-django.readthedocs.io/en/latest/database.html#enabling-database-access-in-tests

This works, but if an ImproperlyConfigured or OperationError happens for another reason, the cache will not be created and the error will be silenced.

Reply:
You may raise a customized [warning](https://docs.python.org/3.7/library/warnings.html).